Source/WebKit/UIProcess/Extensions/WebExtensionMatchPattern.h

```diff
@@ -77,6 +79,9 @@ class WebExtensionMatchPattern : public API::ObjectImpl<API::Object::Type::WebEx
 
     static void registerCustomURLScheme(String);
 
+    static void matchPatternStrings(const MatchPatternSet&, HashSet<String>&);
```

Comment:
Open to better name suggestions for these helper methods

Reply:
This can return the result, as `HashSet<String>&&`. Maybe call this `HashSet<String>&& toStrings(const MatchPatternSet&)`.